db/migrate/20151021105451_create_prisons.rb

```diff
@@ -1,6 +1,6 @@
 class CreatePrisons < ActiveRecord::Migration
   def change
-    create_table :prisons do |t|
+    create_table :prisons, id: :uuid do |t|
```

Comment:
After the hassle that has been caused by changing migrations on the old app, I'm thinking that we should adopt an absolute rule of never changing a migration. Since there is no real data yet, this could be achieved by dropping the old tables and creating them afresh. I realise that it probably seems like unnecessary hassle in this case, but I think it's good hygiene.